commit 7749599d737d205a88bbb6fa755ba095d9b581fa
Author: Gert Wollny <gert.wollny@collabora.com>
Date:   Mon Aug 15 17:15:43 2022 +0200

    r600/sfn: Initialize out buffer when printing op
    
    79ca456b4837b3bc21cf9ef3c03c505c4b4909f6
       r600/sfn: rewrite NIR backend
    
    Closes: #7021
    
    Signed-off-by: Gert Wollny <gert.wollny@collabora.com>
    Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/18130>

diff --git a/src/gallium/drivers/r600/sfn/sfn_instr_export.cpp b/src/gallium/drivers/r600/sfn/sfn_instr_export.cpp
index 3d40ea1796a..00826ed6457 100644
--- a/src/gallium/drivers/r600/sfn/sfn_instr_export.cpp
+++ b/src/gallium/drivers/r600/sfn/sfn_instr_export.cpp
@@ -206,7 +206,7 @@ bool WriteScratchInstr::do_ready() const
 
 void WriteScratchInstr::do_print(std::ostream& os) const
 {
-   char buf[6];
+   char buf[6] = {0};
 
    os << "WRITE_SCRATCH ";
    if (m_address)
